.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```

Comment:
Shouldn't this be Tiptay?

Reply:
Updated `pythonpackage.yml` -> `tiptapy.yml`